UnityProject/Assets/Scripts/Health/Living/HealthSystem.cs

```diff
@@ -746,14 +783,91 @@ public void UpdateClientBodyPartStats(BodyPartType bodyPartType, float bruteDama
 			{
 				//	Logger.Log($"Update stats for {gameObject.name} body part {bodyPartType.ToString()} BruteDmg: {bruteDamage} BurnDamage: {burnDamage}", Category.Health);
 
-				bodyPart.UpdateClientBodyPartStat(bruteDamage, burnDamage);
+				// bodyPart.UpdateClientBodyPartStat(bruteDamage, burnDamage);
 			}
 		}
+		#endregion
 
-		/// ---------------------------
-		/// MISC Functions:
-		/// ---------------------------
+		#region Event methods
```

Comment:
I don't know if it's a good idea to put OnStartServer(), OnStartClient() and OnSpawnServer() in the bottom of the class. I can understand the want to group up methods, but it can be sort of confusing for a person looking in this class for the first time and not seeing any Awake() / OnEnable() / OnStartServer() and so on right after all the variables.

Reply:
I can move them to the top. Maybe consider those as init methods or do their own category, “unity events”